api/handlers.go

```diff
@@ -13,24 +14,39 @@ type Response struct {
 	code    int
 }
 
-// GetSnippet - Handler gets a single code snippet from service by its id
+const (
+	statusError = "error"
+	statusOK = "ok"
```

Comment:
Yeah I agree and usually do the same, but the convention is different in go (which threw me at first as well). because go exports using a capital letter the convention is the same as exporting. If you want the const exported, cap the first letter otherwise just camel case it. An example is the http package consts for status code - http.StatusBadRequest